kivy3/cameras/camera.py

```diff
@@ -39,9 +39,11 @@
     AliasProperty
 from kivy.graphics.transformation import Matrix
 from ..math.vectors import Vector3
+from ..core.object3d import Object3D
+from math import radians
 
 
-class Camera(EventDispatcher):
+class Camera(Object3D, EventDispatcher):
```

Comment:
Object3D is inherited from EventDispatcher itself, so there's no need in double inheritance, we just need to  derive from Object3D.